lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -672,6 +666,22 @@ private function determineIdGeneratorStrategy(AbstractPlatform $platform): int
         return ClassMetadata::GENERATOR_TYPE_TABLE;
     }
 
+    private function truncateSequenceName(string $schemaElementName): string
```

Comment:
Codecov has multiple lines marked as uncovered for `truncateSequenceName()`. Are these lines relevant?

Reply:
Apparently, we're not testing sequences on Oracle. 🤔

Reply:
We do test part of this method though, so there must be existing test that we might be enable for Oracle. Let's `var_dump(debug_print_backtrace());die();` to figure this out :)

Reply:
Apparently it's PostgreSQL (not suprised), but I couldn't get the name of the current test. Let's use `--debug`.

Reply:
Ok so it's `Doctrine\Tests\ORM\Cache\DefaultCollectionHydratorTest::testImplementsCollectionEntryStructure` :thinking: It does use a `City` with a `GeneratedValue`, but the end goal does not really seem to be testing sequences.

Reply:
Apparently, we're not testing ~sequences~ anything on Oracle

FTFY :stuck_out_tongue: